src/rpc/rpcmapper.rs

```diff
@@ -296,22 +296,33 @@ mod tests {
     use super::*;
     use bytes::{Buf, BufMut};
     use cloudevents::{Event, EventBuilder, EventBuilderV10};
+    use protobuf::MessageField;
 
     use crate::cloudevents::CloudEvent as CloudEventProto;
-    use crate::uprotocol::UMessageType;
+    use crate::uprotocol::{UMessage, UMessageType, UPayload};
 
     fn build_status_response(code: UCode, msg: &str) -> RpcClientResult {
-        let status = UStatus::fail_with_code(code, msg);
+        let status: UStatus = UStatus::fail_with_code(code, msg);
         let any = RpcMapper::pack_any(&status)?;
-        Ok(any.try_into().unwrap())
+        let payload =
```

Comment:
Again, funny we both did this haha

Kai gave some good feedback on that PR on how to handle this nicely, perhaps you could [reference that](https://github.com/eclipse-uprotocol/up-rust/pull/27/files#diff-8d66726b55938f3831ad9ba37cb66af6c321e6091a883099cde61b586056cbc3R130-R136)?